packages/debugger/lib/data/sagas/index.js

```diff
@@ -1082,6 +1083,8 @@ export function* decode(definition, ref, compilationId) {
       case "code":
         response = yield* evm.requestCode(request.address);
         break;
+      case "ens":
```

Comment:
Similarly to my point about the identifier name, I think this key should be more explicit, like `"ens-primary-name"` or such. (Yes, I prefer kebab-case ~~everywhere I can get away with it~~ here.)

Reply:
If we ever need to do forward resolution here then you'll thank me ;)